.github/workflows/publish_nightly.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Nightly
+on:
+  schedule:
+    - cron: '0 6 * * *' # Daily at 6AM UTC, 10PM PT
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '17.x'
+      - name: Install dependencies
+        run: npm install
+      - name: Run linter
+        run: npm run lint:js
+      - name: Build
+        run: npm run build
+      - name: Retrieve old Nightly id
+        run: |
+          TAG_ID=`curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/mozilla/firefox-translations/releases/tags/nightly | python3 -c "import sys, json; print(json.load(sys.stdin)['id'])"`
+          echo $TAG_ID
+          echo "TAG_ID=$TAG_ID" >> $GITHUB_ENV
+        continue-on-error: true
```

Comment:
That won't happen because if the deletion steps fail for some reason, the create-release step subsequent to this will also fail because the release is already existent and the action can't overwrite it.

Reply:
I did this so the first one could he properly created